contentcuration/contentcuration/assets/channelList/views/Channel/ChannelModal.vue

```diff
@@ -0,0 +1,705 @@
+          <ThumbnailUpload v-model="thumbnail" :readonly="!canEdit" />
+        </VFlex>
+        <VFlex xs12 sm12 md8>
+          <VAutocomplete
+            v-model="language"
+            :items="languages"
+            prependInnerIcon="language"
+            :readonly="!canEdit"
+            itemText="native_name"
+            itemValue="id"
+          />
+          <VTextField
+            v-model="name"
+            :label="$tr('channelName')"
+            :placeholder="$tr('channelNamePlaceholder')"
+            :readonly="!canEdit"
+            :rules="[() => name.length ? true : $tr('channelError')]"
+          />
+          <VTextarea
+            v-model="description"
+            :label="$tr('channelDescription')"
+            :placeholder="$tr('channelDescriptionPlaceholder')"
+            :readonly="!canEdit"
+            maxlength="400"
+            rows="4"
+            counter
+          />
+          <VCardText v-if="channel.created">
+            {{ $tr('created', { date: new Date(channel.created) }) }}
+          </VCardText>
+          <VCardText v-if="channel.last_published">
+            {{ $tr('published', { date: new Date(channel.last_published) }) }}
+          </VCardText>
+          <template v-if="channel.edit">
+            <p>{{ $tr('deletePrompt') }}</p>
+            <VBtn class="upper" color="error" @click="deleteChannel">
+              {{ $tr('deleteChannel') }}
+            </VBtn>
+          </template>
+        </VFlex>
+        <VFlex xs1>
+          <ChannelStar
+            :channel="channel"
+          />
+        </VFlex>
+      </VLayout>
+      <VLayout v-if="channelDetails && channelDetails.resource_count" row wrap justify-center>
+        <VFlex xs4>
+          <VMenu offset-y>
+            <template v-slot:activator="{ on }">
+              <VBtn
+                color="primary"
+                dark
+                v-on="on"
+              >
+                {{ $tr('downloadReport') }}
+              </VBtn>
+            </template>
+            <VList>
+              <VListTile
+                v-for="(item, index) in downloadOptions"
+                :key="index"
+                :href="item.href"
+                download
+              >
+                <VListTileTitle>{{ item.title }}</VListTileTitle>
+              </VListTile>
+            </VList>
+          </VMenu>
+        </VFlex>
+      </VLayout>
+      <VLayout v-if="channelDetails && channelDetails.resource_count" row wrap>
+        <VFlex xs12 sm12 md8 justifyCenter>
+          <VTabs>
+            <VTab :key="0">
+              {{ $tr('whats_inside') }}
+            </VTab>
+            <VTabItem :key="0">
+              <VLayout row wrap>
+                <VFlex v-if="countBar" xs6>
+                  {{ $tr('resource_size') }}
+                  <span id="count_gauge" data-toggle="tooltip" data-placement="top" :title="$tr('resource_count', { count: channelDetails.resource_count}) + '-' + countBar.text">
```

Comment:
You can probably switch to using [v-tooltip](https://vuetifyjs.com/en/components/tooltips) instead and get rid of the bootstrap tooltip logic here

Reply:
Oh yep, more cleanup needed!